README.md

```diff
@@ -62,6 +62,41 @@ cargo make test-all
 
 Some of the functions in this project are computationally intensive and may take a significant amount of time to compile and complete during testing. To ensure optimal results we use the `make test` command. It enables the running of tests in release mode and using specific configurations replicates the test conditions of the development mode and verifies all debug assertions. For more information refer to the [Makefile.toml](Makefile.toml) for the specific commands and configurations that have been chosen.
 
+## Documentation
+
```

Comment:
I would probably add a sentence (or two) here about where the docs are (i.e., link to the docs site). I think this is what a user would expect to see when they come to the `Documentation` section (before they get into how to run docs locally).